src/test/scripts/get_product_lts_version.py

```diff
@@ -0,0 +1,61 @@
+import sys
+import urllib.request
+import json
+
+known_lts_version = {
+	'jira-software': '8.13.8',
+	'confluence': '7.12.2',
+	'stash': '7.12.1',
+}
+
+
+def get_lts_version(argv):
+	product = argv[0].lower()
+	if product == 'jira':
+		product = 'jira-software'
+	elif product == 'bitbucket':
+		product = 'stash'
+
+	if product in known_lts_version:
+		url = f"https://my.atlassian.com/download/feeds/archived/{product}.json"
+
+		try:
+			fl = urllib.request.urlopen(url)
+			fdata = fl.read()
+			jsdata = json.loads(fdata[10:len(fdata)-1].decode("utf-8"))
+			enterprise_edition = [x for x in jsdata if x['edition'].lower() == 'enterprise']
+			sortedData = sorted(enterprise_edition, key=lambda k:cversion(k['version']), reverse=True)
+
+			if len(sortedData) > 0:
+				lts_version = sortedData[0]['version']
+			else:
+				lts_version = known_lts_version[product]
+
+			# as currently latest lts versoin of bitbucket and confluence don't support k8s
+			# we use none-lts version of those products in the test
+			if cversion(lts_version) < cversion(known_lts_version[product]):
+				lts_version = known_lts_version[product]
+		except:
+			lts_version = known_lts_version[product]
+
+		lts_version = f"{lts_version}-jdk11"
+	else:
+		lts_version = 'unknown'
+
+	return lts_version
+
+
+def cversion(version):
+	vers = [int(v) for v in version.split(".")]
+	mapped_ver = ''
+	for i in range(max(len(vers)-1, 4)):
+		if len(vers) > i:
+			mapped_ver = f'{mapped_ver}{str(vers[i]).zfill(5)}'
```

Comment:
Its not very clear what this expression (`f'{mapped_ver}{str(vers[i]).zfill(5)}'`) is doing, could we extract it into its owned named function?

Reply:
zfill(n) method in python add zeros to the left side of a number to make the length of string n (like padLeft). The purpose of this is making the version an unified string comparable
'7.12.1' => '00007000120000100000'
In this case we can apply string comparison in version and sort the list based on version correctly.

Reply:
Wouldnt it be better to pull in a library to handle version parsing and comparison, this looks useful:
https://pypi.org/project/semantic-version/

Reply:
packaging library is additional module dependency and we need to install in bamboo spec at some point. I preferred to not go to that direction and instead add a simple local method to do the job.